src/ui/panels/quests.js

```diff
@@ -188,27 +196,29 @@ export class Quests {
             for (var enemy in enemies) {
                 let enemiesKilled = characterData.getEnemiesKilled(level, enemy);
 
-                for (
-                    var tier = 1;
-                    tier <=
-                    characterData.calcQuestTier(enemiesKilled, scene.questAmounts[enemy]);
-                    tier++
-                ) {
+                let questTier = characterData.calcQuestTier(
```

Comment:
While we're adding new code we should try to be more functional and only use `let` when we're actually changing the value and `const` otherwise

Reply:
Changed some of them